install.sh

```diff
@@ -102,66 +115,46 @@ function install_local_libs {
-    echo "Installation of OpenRCT2 assumes you have homebrew and use it to install packages."
-
-    echo "Check if brew is installed"
-    package_command="brew"
-    which -s brew
-    if [ $? -eq 1 ]; then
-        echo "brew is not installed, or is not in your \$PATH"
-        echo "Check if MacPorts is installed"
-        which -s port
-        if [ $? -eq 1 ]; then
-            echo "MacPorts not found either, abort"
-            exit
-        else
-            echo "MacPorts found"
-            package_command="sudo port"
-        fi
-    else
-        echo "brew was found"
-    fi
-
-    # Install packages with whatever command was found.
-    # Very possible I'm missing some dependencies here.
-    eval "$package_command install cmake wine"
-
-    if [[ ! -d /usr/include/wine ]]; then
-        # This will almost certainly break as brew changes. Better ideas
-        # welcome.
-        wine_path="/usr/local/Cellar/wine/1.6.2/include/wine"
-        if [ $package_command == "sudo port" ]; then
-            wine_path="/opt/local/include/wine"
-        fi
-        sudo ln -s $wine_path /usr/include
-    fi
-
-    mingw_name=mingw-w32-bin_i686-darwin
-    mingw_tar=$mingw_name"_20130531".tar.bz2
-    mingw_path=/usr/local/$mingw_name
-    if [[ ! -f $cachedir/$mingw_tar ]]; then
-        download "https://downloads.sourceforge.net/project/mingw-w64/Toolchains targetting Win32/Automated Builds/$mingw_tar" $cachedir/$mingw_tar
-    fi
-    if [[ ! -d "$mingw_path" ]]; then
-
-        pushd /usr/local/
-            sudo mkdir $mingw_name
-        popd
-
-        echo "Extracting contents of $mingw_tar to $mingw_path"
-        echo "Don't forget to add $mingw_path/bin to your PATH variable!"
-        sudo tar -xyf $cachedir/$mingw_tar -C $mingw_path
-
-        pushd /usr/local
-            sudo chmod 755 $mingw_name
-            pushd $mingw_name
-                sudo find . -type d -exec chmod 755 {} \;
-            popd
-        popd
-    fi
+function os_x_install_mingw_32 {
+	local mingw_name="mingw-w32-bin_i686-darwin"
+	local mingw_tar="${mingw_name}_20130531.tar.bz2"
+	local mingw_path="/usr/local/$mingw_name"
+
+	if [[ ! -f "$cachedir/$mingw_tar" ]]; then
+		download "https://downloads.sourceforge.net/project/mingw-w64/Toolchains targetting Win32/Automated Builds/$mingw_tar" "$cachedir/$mingw_tar"
+	fi
+
+	if [[ ! -d "$mingw_path" ]]; then
+		echo "Extracting contents of $mingw_tar to $mingw_path"
+		echo "Don't forget to add $mingw_path/bin to your PATH variable!"
+
+		mkdir "$mingw_path"
+		tar -xyf "$cachedir/$mingw_tar" -C "$mingw_path"
+
+		pushd "$mingw_path"
+			find . -type d -exec chmod 755 {} \;
+		popd
+	fi
+}
+
+echo "HOST = $(uname)"
+echo "TARGET = $TARGET"
+
+if [[ "$(uname)" == "Darwin" ]]; then
+	if ! has_cmd "brew"; then
+		echo "Homebrew is not installed, or brew is not in your \$PATH"
+		echo "install instructions: http://brew.sh/"
+		exit 1
+	fi
+
+	# Very possible I'm missing some dependencies here.
```

Comment:
I don't think it's the case, as was proven by successful builds. You can probably remove this comment now.